server/migrations/20230804034846-migrate-emoji-in-document-title.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+const path = require("path");
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    if (process.env.DEPLOYMENT === "hosted") {
+      return;
+    }
+
+    if (process.env.NODE_ENV === "test") {
+      return;
+    }
+
+    const rootDir = "build";
+
+    const { Document } = require(path.join(
+      process.cwd(),
+      rootDir,
+      "server/models"
+    ));
+    const { default: parseTitle } = require(path.join(
+      process.cwd(),
+      rootDir,
+      "shared/utils/parseTitle"
+    ));
+
+    await Document.scope(["withoutState", "withDrafts"]).findAllInBatches(
+      {
+        limit: 100,
+        offset: 0,
+        order: [["createdAt", "ASC"]],
+        paranoid: false,
+      },
```

Comment:
This is safe, models are initialized _before_ the sequelize context is passed on to `umzug` for running migrations.